package.json

```diff
@@ -10,10 +10,10 @@
   },
   "dependencies": {
     "@tailwindcss/postcss7-compat": "^2.0.3",
+    "@videsk/humanize-date": "^1.0.0",
```

Comment:
We create [this library](https://github.com/videsk/humanize-date) to parse dates to human readable.